File: forlab/ForLab.Web/ClientApp/src/@core/models/CMS/ContactInfo.ts
import { BaseEntityDto } from '../common/BaseEntityDto';

export class ContactInfoDto extends BaseEntityDto {
  phone: string = null;
  email: string = null;
  address: string = null;
  latitude: string = null;
  longitude: string = null;
  // Social Links
  facebook: string = null;
  twitter: string = null;
  linkedIn: string = null;
}

File: forlab/ForLab.Data/DbModels/CMSSchema/ContactInfo.cs
using ForLab.Data.BaseModeling;
using System.ComponentModel.DataAnnotations.Schema;

namespace ForLab.Data.DbModels.CMSSchema
{
    [Table("ContactInfos", Schema = "CMS")]
    public class ContactInfo : NullableBaseEntity
    {
        public string Phone { get; set; }
        public string Email { get; set; }
        public string Address { get; set; }
        public string Latitude { get; set; }
        public string Longitude { get; set; }
        // Social Links
        public string Facebook { get; set; }
        public string Twitter { get; set; }
        public string LinkedIn { get; set; }
    }
}
